jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -135,6 +135,12 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
 
 AC_DEFUN_ONCE([OPENJDK_VERSION_DETAILS],
 [
+  # Source the closed version numbers
+  . $AUTOCONF_DIR/../../jdk/make/closed/autoconf/closed-version-numbers
```

Comment:
Can we shorten that path to: `$SRC_ROOT/jdk/make/closed/autoconf/closed-version-numbers`?